### DISCUSSION TITLE
Sync light/dark mode preference across studio and console

### DISCUSSION THREAD

**Discussion by amacneil** [foxglove/community#455] [Q&A]
Foxglove Studio and Console currently have independent light/dark modes. We should sync this state (at least when using the web version of studio).

Perhaps this is also the time to make "follow system" the default across both tools. We should do that after new 3D panel reaches GA.

**Comment by jtbandes** [comment]
For whoever works on this, reminder that the place to store this kind of info in Studio is the User Profile: https://github.com/foxglove/studio/blob/main/packages/studio-base/src/providers/UserProfileLocalStorageProvider.tsx

https://github.com/foxglove/studio/blob/2720aa2036545954b982d3e84b9bf77069130138/packages/studio-base/src/context/UserProfileStorageContext.ts#L9-L12

Right now it’s stored in localStorage only; we’d probably want to keep the localStorage piece as a cache when we add syncing, so the app doesn’t flash the wrong color scheme every time you load it.
